js_modules/dagit/packages/core/.storybook/preview.js

```diff
@@ -13,6 +15,8 @@ import {MemoryRouter} from 'react-router-dom';
 
 import {createGlobalStyle} from 'styled-components/macro';
 
+import '../src/app/blueprint.css';
```

Comment:
Hmm is this a local file? I worry we may have to keep this in sync with the blueprint release used inside dagster-ui in order to avoid subtle problems. Any way to import it from dagster-ui in an assets dir maybe?

Reply:
`@dagster-io/ui` already specifies Blueprint pieces as peer deps, so consumers will need to maintain a sync with the `ui` Blueprint version anyway.

Also the `blueprint.css` file is just importing stuff from `@blueprintjs`, it doesn't copy-paste rules or anything. The file just exists to have a place to house `@import`s.